ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -442,7 +442,7 @@ else
 			then
 				# Start UAVCAN firmware update server and dynamic node ID allocation server.
 				uavcan start fw
-
+				tune_control play -t 1
```

Comment:
This will play the startup tune twice in some setups. Is the intention to make sure it's played over UAVCAN?

Reply:
> This will play the startup tune twice in some setups. Is the intention to make sure it's played over UAVCAN?

Yes. As you said, i  find “legacy” GPS plays the startup tune twice when i connect CAN GPS and “legacy” GPS.
Could you have ideas to make sure it's played over UAVCAN?

Reply:
I'll take a look, we either need to move the regular startup tune later or handle it differently in UAVCAN so it isn't missed.